core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -36,14 +36,15 @@ import scala.collection.Seq
  */
 abstract class BaseConsumerTest extends AbstractConsumerTest {
 
-  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
-  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
-  def testSimpleConsumption(quorum: String): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testSimpleConsumption(quorum: String, groupProtocol: String): Unit = {
     val numRecords = 10000
     val producer = createProducer()
     val startingTimestamp = System.currentTimeMillis()
     sendRecords(producer, numRecords, tp, startingTimestamp = startingTimestamp)
 
+    this.consumerConfig.setProperty(ConsumerConfig.GROUP_PROTOCOL_CONFIG, groupProtocol)
```

Comment:
Adding this change to all the tests is a bit annoying. Have you considered adding this to `IntegrationTestHarness.doSetup` or in `createConsumer`? We could infer it like we did with `isNewGroupCoordinatorEnabled()` in the same class.

Reply:
I did consider that. I have already endured the annoyance. I'll take a look.

Reply:
Done

Reply:
Can we remove this line now?